calico_containers/tests/unit/datastore_test.py

```diff
@@ -126,6 +126,74 @@ def test_pprint(self):
                      rule4.pprint())
 
 
+class TestRules(unittest.TestCase):
+
+    def test_rules(self):
+        """
+        Create a detailed set of rules, convert from and to json and compare
+        the results.
+        """
+        rules_json = """
+{
```

Comment:
it doesn't, but rather than indent it all it would probably be better to add a constant and move to the top